src/app/(project)/projects/[project]/customers/page.tsx

```diff
@@ -179,25 +180,31 @@ export default function CustomersPage() {
                   columnHelper.display({
                     id: 'options',
                     header: () => <></>,
-                    cell: (info) => (
-                      <PopoverBox
-                        triggerButton={
-                          <div className="px-2 py-1 hover:rounded-md hover:bg-stone-800 max-w-min">
-                            <Icon.More />
-                          </div>
-                        }
-                        elements={[
-                          <Link
-                            key="delete_project"
-                            className="flex gap-2 items-center"
-                            href={`/projects/${project?.id}/customers/${info.row.original.id}/delete`}
-                          >
-                            <Icon.Delete stroke="stroke-red-500" />
-                            <span className="text-red-500">Remove</span>
-                          </Link>,
-                        ]}
-                      />
-                    ),
+                    cell: (info) => {
+                      const customer = info.row.original;
+                      return (
+                        <PopoverBox
+                          triggerButton={
+                            <div className="px-2 py-1 hover:rounded-md hover:bg-stone-800 max-w-min">
+                              <Icon.More />
+                            </div>
+                          }
+                          elements={[
+                            <Copy key="copy_id" copyString={customer.wallet}>
+                              <span>Copy Wallet Address</span>
+                            </Copy>,
```

Comment:
Wallets in an array so the copy-paste need to be when the user presses the icon of the wallet in the table item.